plugins/woocommerce/tests/e2e-pw/utils/themes.js

```diff
@@ -0,0 +1,16 @@
+const { exec } = require( 'node:child_process' );
+
+export const activateTheme = ( themeName ) => {
+	return new Promise( ( resolve, reject ) => {
+		const command = `wp-env run tests-cli wp theme activate ${ themeName }`;
```

Comment:
Howdy friends, just a quick note that using `wp-env run tests-cli` like this makes it really hard to port these tests to QIT, as we use these same tests but we are outside of the monorepo.

Ideally, `activateTheme` would use the UI to activate a given theme.

We monkey patched this scenario when syncing the tests, but just wanted to share it here as a JFYI to use the UI if possible in similar scenarios. Ty!

Reply:
For additional context, the feature flag endpoints introduced in `test-helper-apis.php` are fine for us - what really threw us off was coupling with the monorepo environment setup in `activateTheme`

Reply:
@Luc45 why not just copy this helper to your project? 

I think using UI for setup and teardown in e2e and fixture type tasks is an unnecessary slowdown to running e2e and we are not interested in testing theme switching.

Reply:
@samueljseay Replied your DM with details, just a couple of high-level points for context here:
- I agree with the assessment of using the UI being slow
- Using WP CLI is fine, I proposed a minor tweak in how we do it (in DM)